server/container/guice/cassandra-guice/src/main/resources/logback.xml

```diff
@@ -18,5 +18,6 @@
 
         <logger name="org.apache.james" level="INFO"/>
 
+        <logger name="com.datastax.driver.core.QueryLogger.SLOW" level="DEBUG" />
```

Comment:
Same than here:https://github.com/linagora/james-project/pull/642/files#r101797050, should I put the line in some other logback.xml ?